src/client/src/apps/MainRoute/widgets/spotTile/model/spotTileData.ts

```diff
@@ -20,9 +20,12 @@ export interface SpotTileData {
   rfqReceivedTime: number | null
   rfqTimeout: number | null
   rfqPrice: SpotPriceTick | null
-  notional: number
+  notional?: number
 }
 
+type SpotTileDataNotional = Pick<SpotTileData, 'notional'>
+export type SpotTileDataWithNotional = SpotTileData & Required<SpotTileDataNotional>
```

Comment:
see [this](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1284/files#r344925453) explanation